### DISCUSSION TITLE
can i use aqtinstall without the Internet?

### DISCUSSION THREAD

**Discussion by arturon444** [miurahr/aqtinstall#836] [Q&A]
in my office i have qt mirror on local server. can i use aqtinstall without the Internet?

**Comment by pzhlkj6612** [comment]
Hi. Unfortunately this feature has not been implemented yet. You can have a look at:

- https://github.com/miurahr/aqtinstall/issues/224

**Reply by arturon444** [reply]
do you have any suggestions how to make it?

**Reply by pzhlkj6612** [reply]
Well, no. You may have to read and modify the code by yourself.

In addition, in that thread someone mentioned "aqt v2.0.6" (https://github.com/miurahr/aqtinstall/issues/224#issuecomment-1559715145). I think you can give it a try.
